app/controllers/flyover_comments/comments_controller.rb

```diff
@@ -63,6 +71,10 @@ def load_commentable
       end
     end
 
+    def authorize_flyover_comment_show!
+      raise "User isn't allowed to view comment" unless can_view_flyover_comment?(@comment, send(FlyoverComments.current_user_method.to_sym))
```

Comment:
I think this is fine for now. If we run into the case where we're showing the whole thing instead of just loading replies, we can deal with it then.